packages/gcp/data_stream/firestore/_dev/deploy/tf/main.tf

```
@@ -0,0 +1,33 @@
+variable "TEST_RUN_ID" {
+  default = "detached"
+}
+
+variable "project_id" {
+  type    = string
+  default = "elastic-obs-integrations-dev"
+}
+
+variable "collection_name" {
+  type    = string
+  default = "integration-test-collection"
+}
+
+variable "zone" {
+  type = string
+  // NOTE: if you change this value you **must** change it also for test
+  // configuration, otherwise the tests will not be able to find metrics in
+  // the specified region
+  default = "us-central1-a"
+  # https://cloud.google.com/compute/docs/regions-zones#available
+}
+
+provider "google" {
+  project = var.project_id
```

Follows https://github.com/elastic/integrations/pull/2704/files#r831894335

```suggestion
  project = var.gcp_project_id
```